hyou/worksheet.py

```diff
@@ -46,7 +46,8 @@ def refresh(self, entry=None):
             else:
                 raise exception.HyouRuntimeError('The sheet has been removed.')
 
-    def view(self, start_row=None, end_row=None, start_col=None, end_col=None):
+    def view(self, start_row=None, end_row=None,
+             start_col=None, end_col=None, **kwargs):
```

Comment:
Maybe also add the three new argument as keyword arguments here. In my opinion `**kwargs` gives less information and it's not strictly necessary as nothing is going to be done with the other kwargs that aren't those three.